lib/hammer_cli_foreman/compute_resource/ovirt.rb

```diff
@@ -37,6 +37,6 @@ def volume_attributes;
         ]
       end
     end
-    HammerCLIForeman.register_compute_resource('ovirt', Ovirt.new)
+    HammerCLIForeman.register_compute_resource('oVirt', Ovirt.new)
```

Comment:
As it is not sometimes very clear how the compute resources are named we should use the names in lowercase as it was originally.